sdk/include/psdk/shlwapi.h

```diff
@@ -2072,7 +2072,7 @@ BOOL WINAPI IsOS(DWORD);
 typedef struct
 {
     const IID *piid;
-    int        dwOffset;
+    DWORD dwOffset;
```

Comment:
This seems to break compatibility with MS PSDK (at least with the ones for Windows <= 7)...
Does this mean that they actually "fixed" the structure in some newer PSDK? (At least I see that the doc indeed somewhat mentions `DWORD` https://docs.microsoft.com/en-us/windows/win32/api/shlwapi/ns-shlwapi-qitab )

Reply:
It's DWORD in Windows 10 SDK. And the prefix suggest DWORD also. I'll add an #ifdef.